packages/eslint-plugin-office-addins/src/rules/test-for-null-using-isNullObject.ts

```diff
@@ -111,11 +111,11 @@ export = {
         for (let ref = 0; ref < references.length; ref++) {
           const reference = references[ref];
 
-          if (isNullObjectNode(reference.identifier.parent)) {
+          if (isNullObjectNode(reference.identifier.parent as TSESTree.Node)) {
```

Comment:
I think a better way than to explicit cast could be to test the type right?
Like: 
```
if (reference.identifier.parent
&& isNullObjectNode(reference.identifier.parent)) {
```

Reply:
The problem is just a compilation one as a result of change dependency.  The comparison stated complaining because it was say type defined in one package was not the same as a nested package type . . . but the nested package was the same as the higher level.  This just forces them to look at the same package.